android/assets/jsons/translations/template.properties

```diff
@@ -407,6 +407,8 @@ You have entered a Golden Age! =
 [n] sources of [resourceName] revealed, e.g. near [cityName] = 
 A [greatPerson] has been born in [cityName]! = 
 We have encountered [civName]! = 
+[cityStateName] has given us 15 gold as a token of goodwill for meeting us = 
+[cityStateName] has given us 30 gold as we are the first major civ to meet them = 
```

Comment:
I have a feeling that gold given changes with game speed, so these should probably be parameters

Reply:
Just tested this, and turns out they don't. However, when religious city states are added they provide a small amount of faith when meeting them, so I'll parametrize this as a preparation for that